### PR TITLE
Add an autoyast installation schedule file

### PR DESCRIPTION
**Pull request by chcao**
Add an autoyast installation schedule file for power offline migration cases.

- Related ticket: https://progress.opensuse.org/issues/120546#note-12
- Needles: n/a
- Verification run: http://openqa.suse.de/tests/10441920#
- Related mr: https://gitlab.suse.de/coolgw/wegao-test/-/merge_requests/215

### CODE REVIEW

**Review by lemon-suse — APPROVED** [review]

**Comment by tinawang123** [comment]
LGTM

File: schedule/migration/autoyast_offline_pre@pvm.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: autoyast_offline_pre@pvm
+description: >
+  autoYaST installation and prepare base system for offline migration cases
+vars:
+  AUTOYAST_PREPARE_PROFILE: '1'
```
Comment (jknphy) [inline schedule/migration/autoyast_offline_pre@pvm.yaml:6]: we can remove this one.

Comment (jknphy) [reply schedule/migration/autoyast_offline_pre@pvm.yaml:6]: actually textmode is already set:
offline_sles15sp4_basesys-srv_all_spvm_pre set it in 
https://openqa.suse.de/admin/test_suites so there is no need...

Comment (chcao) [reply schedule/migration/autoyast_offline_pre@pvm.yaml]: Thanks for review, I updated it.

**Pull request merged**